mailtrap/models/templates.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+
+from pydantic.dataclasses import dataclass
+
+from mailtrap.models.common import ParametersObject
+
+
+@dataclass
+class CreateEmailTemplateParams(ParametersObject):
+    name: str
+    subject: str
+    category: str
+    body_text: Optional[str] = None
+    body_html: Optional[str] = None
+
+
+@dataclass
+class UpdateEmailTemplateParams(ParametersObject):
```

Comment:
Why do we need 2 models: `CreateEmailTemplateParams` and `UpdateEmailTemplateParams`? Won't a single one be enough?